nym-wallet/src/components/CopyToClipboard.tsx

```diff
@@ -35,6 +35,7 @@ export const CopyToClipboard = ({ text = '', iconButton }: { text?: string; icon
           sx={{
             color: 'text.primary',
           }}
+          disabled={!text}
```

Comment:
Instead of disabling here if `!text` then you could just not render this copy component if text is undefined

Reply:
Yes we could, but I followed what the design suggests